.gitlab/deploy_packages/windows.yml

```diff
@@ -21,6 +21,14 @@ deploy_packages_windows-x64-7:
       --include "datadog-installer-*-1-x86_64.zip"
       --include "datadog-installer-*-1-x86_64.exe"
       $OMNIBUS_PACKAGE_DIR/pipeline-$CI_PIPELINE_ID $S3_RELEASE_ARTIFACTS_URI/msi/x86_64/
+    # TODO: When this snippet is removed, change upload destination in agent-release-management https://github.com/DataDog/agent-release-management/blob/27bcd404709e4e7aaea3b1d5f272fab5be999a57/ci/libraries/msi.sh#L862-L869
```

Comment:
Chatted about this on slack - I don't think we can just drop this section and change agent-release-management without breaking backward compatibility. Let's fix the current issue and then investigate some options for long term support 👍